Dockerfile

```diff
@@ -71,3 +69,5 @@ ENV SKIP_MY_POSTINSTALL=true
 
 # Install code
 COPY . ./
+
+RUN dart pub get
```

Comment:
move `dart pub get` to after `code` is installed so that `/.dart_tool/` files (generated by this step) remain in the container (maybe there is a better way to do this ...)